deployment/terraform/variables.tf

```diff
@@ -21,3 +21,18 @@ variable "ssl_certificate_arn" {}
 variable "cdn_price_class" {
   default = "PriceClass_100"
 }
+
+variable "remote_state_bucket_prefix" {
+  default = "districtbuilder"
+}
+
+variable "is_subdomain" {}
+
+variable "ssh_identity_file_path" {
+  default = "~/.ssh/district-builder.pem"
+}
+
+variable "cloudfront_aliases" {
+  description = "List of CNAMES that will refer to this project"
+  default     = []
+}
```

Comment:
I don't think that CI will work without the encrypted PEM file that was deleted below. We need this to be able to establish SSH connections from Travis.